### ISSUE TITLE
PTVSD manylinux wheels don't pass auditwheel checks

### ISSUE THREAD

**Issue by SylvainCorlay** [microsoft/ptvsd#2119]
## Environment data

- PTVSD version: 4.3.2 (latest)
- OS and version: Manylinux1
- Python version (& distribution if applicable, e.g. Anaconda): Any

## Actual behavior

auditwheel fails on ptvsd's maynlinux wheels with 

```
RuntimeError: Invalid binary wheel, found the following shared library/libraries in purelib folder:
	pydevd_cython.cpython-35m-x86_64-linux-gnu.so
	attach_linux_amd64.so
	attach_linux_x86.so
```

**Comment by int19h** [comment]
I don't think this causes any actual problems, because we load all that code conditionally, so on any platform for which those binaries aren't compiled, the debugger still works as a pure Python library, although functionality may be reduced (e.g. no attach-to-PID).

That said, we should try to fix the warning. This may be tricky because of how we are packaging our vendored copy of pydevd, though.

**Comment by int19h** [comment]
It still applies to debugpy, so moving it to that repo.
